src/pyedb/dotnet/edb_core/siwave.py

```diff
@@ -1488,6 +1489,19 @@ def icepak_use_minimal_comp_defaults(self):
         _, value = cell.GetProductProperty(siwave_id, 422, "")
         return bool(value)
 
+    def create_crosstalk_config_file(self, scan_type="impedance"):
```

Comment:
Can you use a create a another method to create impedance scan config file?

Reply:
it is also supporting impedance scan

Reply:
changed the name to create_impedance_crosstalk_scan